src/providers/connectors/sequence.ts

```diff
@@ -20,7 +21,18 @@ const ConnectToSequence = async (
     }
   }
 
-  const wallet = new sequence.Wallet(opts?.defaultNetwork || 'mainnet');
+  let wallet;
+
+  try {
+    wallet = sequence.getWallet();
+  } catch (err) {
+    let walletConfig = {};
+    if (opts?.networkRpcUrl) {
+      walletConfig = { networkRpcUrl: opts?.networkRpcUrl }
```

Comment:
```suggestion
      walletConfig = { networkRpcUrl: opts.networkRpcUrl }
```

Guaranteed to exist via check above

Reply:
Good catch! Fixed in commit #[89fc860](https://github.com/WalletConnect/web3modal/pull/726/commits/89fc860271df8985bee3c34cdf30e284186591eb)